pkgs/development/web/edge-runtime/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
+, lib
+, callPackage
+, fetchFromGitHub
+, rustPlatform
+, nix-update-script
+, darwin
+, openssl
+, pkg-config
+}:
+
+let
+  pname = "edge-runtime";
+  version = "1.6.6";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
```

Comment:
Thank you for the recommendation, but I'll pass on this one, I don't like rec, because you never know what is actually being used and what is not.

Reply:
Using rec here is not actually a problem unless you are referring to overlays in which case let is not changing the behaviour.

Reply:
I think it makes it hard tuo understand at a glance which attributes are used and which are not. Its a matter of personal preference really. I know this can spark lively debate though, i can understand if you disagree .

Reply:
Are you using nil lsp? That highlights where variables are used.

Reply:
Ah, nice to know.
Im testing out nixd at the moment. Ill keep this in mind